app/models/event.rb

```diff
@@ -62,18 +62,30 @@ def single_player?
   end
 
   # Everything below this is leagues only code and will be moved to Leagues.rb once there is an actual option to create Leagues AND Tourneys, etc.
-
-  def add_test_teams
-    max_teams.times do |index|
-      teams << FactoryBot.create(:team)
+  # Joining a single Player for a single Player Event
+  # This method is only temporary until we have a working teams-infrastructure
+  def add_single_player_team(user)
+    if teams.length < max_teams
+      teams << Team.new(name: "#{user.first_name} #{user.last_name}"  , private: false)
```

Comment:
maybe take the users email as the teamname because that one is assured to be unique, while the name might not be. This is a minor issue though.

Reply:
I would leave it as-is for now, since this code will definitely be removed once the other teams add the joining-functionality. For our PO, this naming will look better during demoing.